active/0000-blueprint-remove-old-files.md

```diff
@@ -0,0 +1,48 @@
+- Start Date: 2016-12-17
+- RFC PR: (leave this empty)
+- Ember CLI Issue: (leave this empty)
+
+# Summary
+
+Give blueprint generators the ability to clean up old files.
+
+# Motivation
+
+We want to eliminate the noise of having old files laying after updating
+ember-cli using `ember init`.
+
+# Detailed design
+
+We'd like a declarative API for blueprints to delete files instead of only
+create. It would be essentially syntactic sugar for removing the file yourself
+in an `afterInstall` hook. It would be an array on the blueprint's `index.js`.
+
+```js
+// ember-cli/bluprints/blah/index.js
+module.exports = {
+  // ...
+
+  oldFilesToRemove: [
```

Comment:
its mostly because it is currently a property on the class, and not the instance. Which is a tad funky (although nice for simple addon mains)